### DISCUSSION TITLE
Wire Quick Reconnect

### DISCUSSION THREAD

**Discussion by bbbbbgdn** [cables-gl/cables_docs#713] [Answers & Questions]
I love the feature that I can reconnect wires with right-clicking it in any place and make it to another output. I wish there was a way to quickly reconnect the wires both ways. Does this make sense? 

![Mar-28-2024 00-58-16](https://github.com/cables-gl/cables_docs/assets/12091461/33e18851-b15c-49f0-b3b6-e48c7ed5ea9e) <img width="282" alt="Screenshot 2024-03-28 at 00 58 42" src="https://github.com/cables-gl/cables_docs/assets/12091461/ac891866-8838-4a8f-a869-9838f4ce6c50">

**Comment by pandrr** [comment]
those are already options in the user preferences

**Comment by bbbbbgdn** [comment]
Hey! Can you point me at one of this? 

I believe there's next in settings:
<img width="266" alt="image" src="https://github.com/cables-gl/cables_docs/assets/12091461/ebecbb38-aba2-43d5-bdf1-76f2f68e41fc">

But I rather voting for an ability to allow for dragging a cable and connecting it either to the input or output, I see now there's only a way to connect it to output after dragging.

I think it makes much sense to have it both ways, doesn't it?

**Comment by bbbbbgdn** [comment]
There's a way to quickly reconnect the lect cable to the blue position, but not the yellow one. Is there a specific reason for this?

<img width="251" alt="cables_proposal" src="https://github.com/cables-gl/cables_docs/assets/12091461/2d4dd262-2c37-4b97-ad8e-ea40f220885d">

**Comment by bbbbbgdn** [comment]
I think I found it, it changes direction depending on the proximity to the Input or Output when you start dragging. It was unclear, and in real scenario, I think it would be much more useful if it make choice not depend on the proximity, rather just keeping possibility to stick to either of the ends, depending on the side I drag it to. 

I thinks visually it's clearer to explain, I talk about this:
<img width="251" alt="cables_proposal)2" src="https://github.com/cables-gl/cables_docs/assets/12091461/4fd153c8-a4ad-43a7-a31a-b2b2960ca7d9">

**Comment by steam0r** [comment]
it depends on where you grab the cable, upper half will disconnect the upper connection, lower half will disconnect lower connection

https://github.com/cables-gl/cables_docs/assets/151271/fb58cf3c-1ddb-4bc9-ad65-d33c6ac7b344

**Reply by steam0r** [reply]
moved this from issues to the discussions

**Reply by bbbbbgdn** [reply]
Yeah, I found it. But based on my experience, in the wildest salads of cables, it's usually isn't easy to measure this halves.

And since I find this feature extremely useful, I think it might be even more useful, if allowed both-ways of connection, depending solely on the side that I connecting it to.
